requirements-full.txt

```diff
@@ -190,7 +190,7 @@ pytrends==4.7.3
 pytz-deprecation-shim==0.1.0.post0; python_version >= "3.6" and python_full_version < "3.0.0" or python_full_version >= "3.6.0" and python_version >= "3.6"
 pytz==2021.3; python_full_version >= "3.6.8" and python_version >= "3.6" and python_full_version < "4.0.0"
 pyupgrade==2.18.2; python_full_version >= "3.6.1"
-pywin32==301; sys_platform == "win32" and platform_python_implementation != "PyPy" and python_version >= "3.6"
+pywin32==302; sys_platform == "win32" and platform_python_implementation != "PyPy" and python_version >= "3.6"
```

Comment:
I keep running into issues with pywin32. I think we should keep it at 301.

Reply:
actually 302 is working again for me now. Windows inconsistencies are annoying.

Reply:
Is this good to go then?

Reply:
It is to me=) @aia ?